### DISCUSSION TITLE
Analyzing PhenoCycler(codex data) with Seurat

### DISCUSSION THREAD

**Discussion by MaayanPour** [satijalab/seurat#6775] [General]
Using this https://satijalab.org/seurat/articles/spatial_vignette_2.html 
analysis framework for codex data, the structure of the data required in order to view the spacial analysis (ImageDimPlot, for example) at the example on the website, is the output of the old CODEX software. I wonder how to use this framework for the new CODEX data (Phenocycler), and specifically the output from HALO. Thanks

**Comment by blakesells7** [comment]
I am also having this difficulty. Is there a way to directly output Halo object data into Seurat?
